### ISSUE TITLE
`labeling 0.1.13 requires pillow<10, but you have pillow 10.0.1 which is incompatible.`

### ISSUE THREAD

**Issue by paulmenzel** [indico/indico#5988]
```
$ git log --oneline --no-decorate -1
aeedf0fff2 Apply DS to older fixes, improve color contrast (#5980)
$ pip install -e '.[dev]'
[…]
ERROR: pip's dependency resolver does not currently take into account all the packages that are installed. This behaviour is the source of the following dependency conflicts.
labeling 0.1.13 requires pillow<10, but you have pillow 10.0.1 which is incompatible.
napari 0.4.18 requires sphinx<5,>=4.3.0, but you have sphinx 6.2.1 which is incompatible.
pyabc 0.12.12 requires sqlalchemy>=2.0.12, but you have sqlalchemy 1.4.49 which is incompatible.
python-lsp-server 1.7.4 requires jedi<0.19.0,>=0.17.2, but you have jedi 0.19.0 which is incompatible.
spyder 5.4.4 requires jedi<0.19.0,>=0.17.2, but you have jedi 0.19.0 which is incompatible.
[…]
$ git grep pillow requirements.*
requirements.in:pillow
requirements.txt:pillow==10.0.1
```

It’d be great if you could update the requirements on the master branch. (I remember you said, you do it before releases, but to test the development version, more updates here would be great.)

**Comment by ThiefMaster** [comment]
I have a dependency update included in #5978 which I will merge once Celery fixed 3.12 support. I could easily move it into a separate PR though in case they take longer than I hope they do.

**Comment by ThiefMaster** [comment]
Actually, nevermind, Pillow 10.0.1 is the libwebp security fix. This is part of the 3.2.x branch which I'm just about to merge back into master anyway.
